pkg/source/source_test.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package source_test
 
 import (
+	"context"
```

Comment:
Do we need tests that check whether `Start` will return an error if the passed context is cancelled?

Reply:
done